docs/migrate-from-sidecar-to-ambient/migration.adoc

```diff
@@ -0,0 +1,2114 @@
+HTTP_STATUS=$(kubectl exec $TEST_POD -n $NAMESPACE -- curl -s -w "%{http_code}" -o /dev/null http://reviews.$NAMESPACE:9080/reviews/1)
+if [[ "$HTTP_STATUS" == "200" ]]; then
+  echo "PASS: Normal requests working through waypoint"
+else
+  echo "FAIL: Normal requests failing: HTTP $HTTP_STATUS"
+  exit 1
+fi
+
+# Test header-based routing (if HTTPRoute is configured)
+if kubectl get httproute reviews -n $NAMESPACE >/dev/null 2>&1; then
+  # Test default routing (should go to reviews-v1)
+  DEFAULT_RESPONSE=$(kubectl exec $TEST_POD -n $NAMESPACE -- curl -s http://reviews.$NAMESPACE:9080/reviews/1)
+  HEADER_RESPONSE=$(kubectl exec $TEST_POD -n $NAMESPACE -- curl -s -H "end-user: jason" http://reviews.$NAMESPACE:9080/reviews/1)
+
+  DEFAULT_VERSION=$(echo "$DEFAULT_RESPONSE" | jq -r '.podname' | grep -o 'reviews-v[0-9]' || echo "unknown")
+  HEADER_VERSION=$(echo "$HEADER_RESPONSE" | jq -r '.podname' | grep -o 'reviews-v[0-9]' || echo "unknown")
+
+  echo "Default version (no header): $DEFAULT_VERSION"
+  echo "Version with 'end-user: jason' header: $HEADER_VERSION"
+
+  if [[ "$DEFAULT_VERSION" == "reviews-v1" && "$HEADER_VERSION" == "reviews-v2" ]]; then
+    echo "PASS: Header-based routing working correctly - default→v1, jason→v2"
+  elif [[ "$DEFAULT_VERSION" == "reviews-v1" && "$HEADER_VERSION" == "reviews-v1" ]]; then
+    echo "FAIL: Header routing not working - both requests go to v1 (waypoint may not be processing traffic)"
+  else
+    echo "WARNING: Routing behavior unclear - default: $DEFAULT_VERSION, header: $HEADER_VERSION"
+  fi
+else
+  echo "INFO: No HTTPRoute configured for reviews service - skipping routing test"
+fi
+
+echo "PASS: Step 7 validation passed - ready to proceed to Step 8"
+----
+
+**If validation fails**: Check waypoint configuration and ensure L7 policies are correctly targeted.
+
+**Rollback Decision Point**: If Step 7 validation fails, disable waypoint usage:
+[source,console]
+----
+# Disable waypoint usage but keep ambient mode
+kubectl label namespace bookinfo istio.io/use-waypoint-
+# L4 policies will still work through ztunnel, L7 policies will be disabled
+----
+**Risk**: High - L7 policy enforcement may be lost. Consider full rollback to sidecar mode if L7 policies are critical.
+
+[[step-8-policy-simplification]]
+=== Step 8: Policy Simplification
+
+[[step-81-remove-redundant-sidecar-policies]]
+==== Step 8.1 Remove Redundant Sidecar Policies
+
+Once traffic flows through waypoints, remove duplicate sidecar-specific policies:
+
+[source,console]
+----
+# List all authorization policies to identify duplicates
+kubectl get authorizationpolicy -A
+# List virtual services to identify those replaced by HTTPRoutes
+kubectl get virtualservice -A
+----
+
+Delete the redundant sidecar policies by running the following commands:
+[source,console]
+----
+# Example: Remove sidecar-specific authorization policies
+kubectl delete authorizationpolicy authorizationpolicy-name -n bookinfo
```

Comment:
What is `authorizationpolicy-name`? 

Probably we should use \<authorizationpolicy-name\> to indicate it requires user-input.